package.json

```diff
@@ -5,31 +5,30 @@
   "description": "",
   "main": "",
   "dependencies": {
-    "@types/node": "8.0.34",
+    "@types/node": "12.0.7",
     "@types/underscore": "1.8.18",
     "@types/webpack": "4.4.32",
     "@types/webpack-dev-server": "3.1.6",
     "blueimp-md5": "2.10.0",
-    "cache-loader": "3.0.1",
+    "cache-loader": "4.0.0",
     "clipboard": "2.0.4",
-    "css-hot-loader": "1.3.9",
-    "css-loader": "0.28.11",
+    "css-loader": "2.1.1",
     "emoji-datasource-google": "4.0.4",
     "emoji-datasource-google-blob": "npm:emoji-datasource-google@3.0.0",
     "emoji-datasource-twitter": "4.0.4",
     "error-stack-parser": "2.0.2",
     "expose-loader": "0.7.5",
-    "file-loader": "1.1.11",
     "flatpickr": "4.5.7",
+    "file-loader": "4.0.0",
```

Comment:
These are out of alphabetical order again, which leads to unnecessary churn later when `yarn` automatically realphabetizes them. Did edit this manually? If so, are you aware of `yarn add` and `yarn upgrade`?

Reply:
We should probably document `yarn add` on `docs/subsystems/dependencies.md`; the JS section of that isn't super well-written and should at least highlight the core commands folks should run.  Maybe also in `front-end-build-process.md`?